src/projection.jl

```diff
@@ -123,7 +115,6 @@ ProjectTo{AbstractArray}(element = ProjectTo{Float64}(), axes = (Base.OneTo(2),
 ProjectTo(::Any) # just to attach docstring
 
 # Generic
-(::ProjectTo{T})(dx::T) where {T} = dx  # not always correct but we have special cases for when it isn't
```

Comment:
Removing this doesn't seem to break anything.

I now think this is only correct for Number and AbstractArray, who do their own stunts. For everyone else, the acceptable thing is a Tangent.